core/src/main/resources/jenkins/install/SetupWizard/wizard-ui.jelly

```diff
@@ -1,13 +1,7 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler">
-  <script type="text/javascript">
-    <j:getStatic var="defaultUpdateSiteId" className="hudson.model.UpdateCenter" field="ID_DEFAULT"/>
-    var defaultUpdateSiteId = '${defaultUpdateSiteId ? defaultUpdateSiteId.replace("'", "") : "default"}';
-    var setupWizardExtensions = [];
-    var onSetupWizardInitialized = function(extension) {
-      setupWizardExtensions.push(extension);
-    };
-  </script>
+  <div id="target-div" data-default-update-site-id="${hudson.model.UpdateCenter.ID_DEFAULT}"></div>
```

Comment:
`target-div` is a bit too generic as for me, I'd suggest something more specific to avoid potential conflicts

Reply:
Is this id name is okay? Because this div will return default site id.
`<div id="default-site-id" data-default-update-site-id="${hudson.model.UpdateCenter.ID_DEFAULT}"></div>`